docs/tutorial.md

```diff
@@ -495,6 +495,44 @@ Your take really. Let's take the former approach:
 
 We're using [`#resolve()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-resolve) to mark a conflict as resolved: it accepts a conflict object, and a resolution one; the latter is what will be updated locally and sent for resynchronization on a next call to [`#sync()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-sync) — which is exactly what we're doing after we've resolved all our conflicts.
 
+## Receiving changes via Websocket
```

Comment:
I suggest we move this to a new dedicated section of the docs, eg. `Advanced usage`. The tutorial is really aimed at beginners with simple, general use cases.